Lib/fontTools/ttLib/__init__.py

```diff
@@ -299,8 +300,11 @@ def saveXML(self, fileOrPath, progress=None, quiet=None,
 			if progress:
 				progress.set(i)
 			tag = tables[i]
-			if splitTables:
+			if splitTables or splitGlyphs:
```

Comment:
when splitGlyphs is True here you're calling tagToIdentifier for all the tables, even though only glyf currently support the extra argument

Reply:
Right.  I can change the condition to if splitTables or (splitGlyphs && (tag == 'glyp')) so that the tagToIdentifier is only called when needed.

Reply:
BTW - splitTables and splitGlyphs work well together - it is actually the use case which we always use.